src/platform.h.in

```diff
@@ -156,8 +156,7 @@
 /* Define to 1 if the system has the type `_Bool'. */
 #undef HAVE__BOOL
 
-/* Define to the sub-directory in which libtool stores uninstalled libraries.
-   */
+/* Define to the sub-directory where libtool stores uninstalled libraries. */
```

Comment:
I had to revert this as autogen replaces it back to previous state based on config/libtool.m4

Reply:
Interesting... not sure what generated the change in the first place :\